consonance-arch/src/main/java/io/consonance/arch/containerProvisioner/ContainerProvisionerThreads.java

```diff
@@ -272,6 +272,9 @@ public Void call() throws Exception {
                             arguments.addAll(Arrays.asList(parse.getArguments()));
                             arguments.add("--instance-types");
                             arguments.add(tempFile.getAbsolutePath());
+                            // I think this needs to be passed in each time
```

Comment:
@briandoconnor 
FYI, tracking these issues in https://github.com/Consonance/consonance/issues/140 

I'm removing this because this seems to be a misunderstanding. `total-nodes-num` determines the total number of nodes that should be deployed and forces youxia into only creates nodes up to that number all of one type. 
So this disables the ability to provision nodes of different instance types.